### DISCUSSION TITLE
Problem bunker silo work

### DISCUSSION THREAD

**Discussion by Tigger86** [Courseplay/Courseplay_FS22#3579] [General]
I have already watched a lot of videos but they are all a year old. And there is no more parking position.

But I set it so that the parking position is the same as the target position.
But it doesn't find the target every time.
Is there a solution or what am I forgetting?

Thanks

**Comment by Tensuko** [comment]
That depends on how your place look like.
If he can't find a way there, there must be too less space.
Have you tried place the position at a bigger place just for testing?
Screenshots can help.
